assets/openobserve/main.py

```
@@ -0,0 +1,96 @@
+        """
+        Removes a previously ingested dataset before ingesting a new one, must not throw error
+        when no dataset was ingested
+        """
+        self.docker_execute("curl -X DELETE -u 'root@clpbench.com:password' localhost:5080/api/default/streams/clpbench1")
+        while True:
+            status = self.docker_execute("""\
+                    curl http://localhost:5080/api/default/clpbench1/_json -i -u 'root@clpbench.com:password' -d '[]' -w "%{http_code}" -s -o /dev/null\
+                    """)
+            if status != "400":
+                break
+            else:
```

Ruff (0.12.2)

Unnecessary `else` after `break` statement

Remove unnecessary `else`

(RET508)